hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/wal/SequenceIdAccounting.java

```diff
@@ -480,6 +480,9 @@ Map<byte[], List<byte[]>> findLower(Map<byte[], Long> sequenceids) {
           continue;
         }
         for (Map.Entry<ImmutableByteArray, Long> me : m.entrySet()) {
+          if (me.getKey().toString().equals("METAFAMILY")) {
```

Comment:
Better add some comments to explain this logic?

Reply:
Sure @Apache9 , just added some comments. Please let me know if those are ok.